godel_process_planning/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(godel_process_planning)
+
+find_package(catkin REQUIRED COMPONENTS
+  descartes_core
+  descartes_moveit
+  descartes_planner
+  descartes_trajectory
+  godel_msgs
+  moveit_ros_planning_interface
+  roscpp
+)
+
+catkin_package(
+  INCLUDE_DIRS include
+  CATKIN_DEPENDS
+    descartes_core
+    descartes_moveit
+    descartes_planner
+    descartes_trajectory
+    godel_msgs 
+    moveit_ros_planning_interface 
+    roscpp
+)
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+include_directories(include)
+include_directories(
```

Comment:
These two lines could be merged into a single _include_directories_ call that adds the local _include_ and catkin directories.